python/src/FunctionalChaosAlgorithm_doc.i.in

```diff
@@ -46,94 +46,66 @@ weights : sequence of float
+The polynomial chaos expansion enables to approximate the output random variable of interest :math:`\vect{Y} = g(\vect{X}) \in \Rset^p` where :math:`\vect{X}` is the input random vector of dimension *n* by the  response surface :
 
-Consider that the output random variable of interest
-:math:`\vect{Y} = g(\vect{X})` has a finite variance, i.e.
-:math:`\Var{g(\vect{X})} < \infty`
+.. math::
 
-Then it is shown that :math:`\vect{Y}` may be expanded onto the PC basis as
-follows:
+    \tilde{\vect{Y}} = \sum_{k \in K} \vect{\alpha}_k \Psi_k \circ T(\vect{X})
 
-.. math::
+where :math:`\vect{\alpha}_k \in \Rset^p`, *T* is an isoprobabilistic transformation which maps the multivariate distribution of :math:`\vect{X}` into the multivariate distribution :math:`\mu = \prod_{i=1}^n \mu_i`, and :math:`(\Psi_k)_{k \in \Nset}` a multivariate polynomial basis of :math:`\cL^2_{\mu}(\Rset^n,\Rset)` which is orthonornal according to the distribution :math:`\mu`. *K* is a finite subset of :math:`\Nset`. 
 
-    g(\vect{X}) = \sum_{j=0}^{\infty} \vect{a}_j \Psi_j(\vect{X})\circ T(\vect{X})
+The distribution :math:`\mu` is supposed to have an independent copula and *Y* be of finite second moment.
 
-where the :math:`\vect{a}_j=(a_{j,1}, \dots, a_{j,n_Y})`'s are
-deterministic vectors of coefficients that fully characterize the response
-:math:`\vect{Y}`. :math:`\Psi_j`'s are multivariate polynomials that are
-orthonormal with respect to the joint probability density function (PDF)
-:math:`f_{\vect{X}}(\vect{x})` of :math:`\vect{X}`. :math:`T` is the
-isoprobabilistic transformation which enables to rescale
-the components :math:`X_i` of random vector :math:`\vect{X}` such as:
+When :math:`\vect{Y}` is of dimension :math:`p>1`,  OpenTURNS operates marginal by marginal, using the same multivariate orthonormal basis :math:`(\Psi_k(\vect{x}))_{k \in \Nset}` for all the marginals. Then, we obtain the following response surfaces for each marginal :math:`Y_i` :
 
 .. math::
 
-    Y \equiv h(\vect{U}) &= g \circ T^{-1}(\vect{U}) \\
-                         &= \sum_{j=0}^{\infty} \vect{a}_j \Psi_j(\vect{U})
+    \tilde{Y}_i = \sum_{k \in K} \alpha_k^i \Psi_k \circ T(\vect{X})
 
-with :math:`\vect{U} = T(\vect{X})`
-
-Consider the case of independent input random variables. The PDF may be cast as:
+The final multivariate response surface is presented as follows :
 
 .. math::
 
-    f_{\vect{X}}(\vect{x}) = \prod_{i=1}^{n_X} f_{X_i}(x_i)
+    \tilde{\vect{Y}} = \sum_{k \in K} \vect{\alpha}_k \Psi_k \circ T(\vect{X})
 
-where :math:`f_{X_i}(x_i)` is the marginal PDF of :math:`X_i`.
-Let us consider a family :math:`\{\pi_j^{(i)}, j \in \Nset\}` of orthonormal
-univariate polynomials with respect to :math:`f_{X_i}`, i.e.:
+where :math:`\vect{\alpha}_k = ( \alpha_k^1, \dots,  \alpha_k^p)`.
 
-.. math::
+We detail here all the steps required in order to create a polynomial chaos algorithm  when the output random variable *Y* is scalar.
```

Comment:
That line (and others) is definetely too long, I have to drag the pointer on github to see it.